CHANGELOG.md

```diff
@@ -24,7 +23,6 @@
 
 ### Bug Fixes
 
-- Fixed issue with Go dependencies caused by removed tag in `openshift/api` repository ([#2466](https://github.com/operator-framework/operator-sdk/issues/2466))
 - Fixed a regression in the `operator-sdk run` command that caused `--local` flags to be ignored ([#2478](https://github.com/operator-framework/operator-sdk/issues/2478))
```

Comment:
It was removed because it was already released at 0.15.1 ( it is a mistake )

Reply:
It's not a mistake. We include the bug fixes that we backport in the patch release that fixes it AND the next minor release. If this is the only substantive change for this PR, I think we can just close it.